docs/csharp/language-reference/keywords/where-generic-type-constraint.md

```diff
@@ -15,44 +15,36 @@ ms.author: "wiwagn"
 
 [!code-csharp[using an interface constraint](../../../../samples/snippets/csharp/keywords/GenericWhereConstraints.cs#2)]
 
-The `where` clause can specify that the type is a `class` or a `struct`. The `struct` constraint removes the need to specify a base class constraint of `System.ValueType`. The `System.ValueType` type may not be used as
-a base class constraint. The following example shouws both the `class` and `struct` constraints:
+The `where` clause can specify that the type is a `class` or a `struct`. The `struct` constraint removes the need to specify a base class constraint of `System.ValueType`. The `System.ValueType` type may not be used as a base class constraint. The following example shows both the `class` and `struct` constraints:
 
 [!code-csharp[using the class and struct constraints](../../../../samples/snippets/csharp/keywords/GenericWhereConstraints.cs#3)]
 
-The `where` clause may also include an `unmanaged` constraint. The unmanaged
-constraint limits the type parameter to types known as **unmanaged types**.
-An **unmanaged type** is a type which is not a reference type and doesn't
-contain reference type fields at any level of nesting. The `unmanaged`
-constraint makes it easier to write low-level interop code in C#. This constraint
-enables reusable routines across all unmanaged types. The `unmanaged` constraint
-cannot be combined with the `class` or `struct` constraint. The `unmanaged` constraint
-implies the `struct` constraint:
+The `where` clause may also include an `unmanaged` constraint. The `unmanaged` constraint limits the type parameter to types known as **unmanaged types**. An **unmanaged type** is a type that isn't a reference type and doesn't contain reference type fields at any level of nesting. The `unmanaged` constraint makes it easier to write low-level interop code in C#. This constraint enables reusable routines across all unmanaged types. The `unmanaged` constraint can't be combined with the `class` or `struct` constraint. The `unmanaged` constraint enforces that the type must be a `struct`:
 
 [!code-csharp[using the unmanaged constraint](../../../../samples/snippets/csharp/keywords/GenericWhereConstraints.cs#4)]
 
-The `where` clause may also include a constructor constraint. It is possible to create an instance of a type parameter using the new operator; however, in order to do so the type parameter must be constrained by the constructor constraint, `new()`. The [new() Constraint](new-constraint.md) lets the compiler know that any type argument supplied must have an accessible parameterless--or default-- constructor. For example:
+The `where` clause may also include a constructor constraint, `new()`. That constraint makes it possible to create an instance of a type parameter using the new operator. The [new() Constraint](new-constraint.md) lets the compiler know that any type argument supplied must have an accessible parameterless--or default-- constructor. For example:
```

Comment:
nit: code-style (backticks) in the second sentence for the operator name: *...using the `new` operator*